@here/harp-materials/lib/ShaderChunks/ExtrusionChunks.ts

```diff
@@ -4,15 +4,22 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+const MIN_BUILDING_HEIGHT_SQUARED = 0.001 * 0.001;
```

Comment:
Why not use MIN_BUILDING_HEIGHT?

Reply:
Circular dependencies :/
This is first iteration, wil ltry to get proper place for definitions like these - it will end-up with "MaterialDefs.ts" or something that will contain only constants shared by decoders and renderer part.

Reply:
Fixed, `MIN_BUILDING_HEIGHT` is now defined once.